clang/lib/Sema/SemaChecking.cpp

```diff
@@ -19185,7 +19185,8 @@ static bool isLayoutCompatible(ASTContext &C, EnumDecl *ED1, EnumDecl *ED2) {
 
 /// Check if two fields are layout-compatible.
 static bool isLayoutCompatible(ASTContext &C, FieldDecl *Field1,
-                               FieldDecl *Field2) {
+                               FieldDecl *Field2,
+                               bool IgnoreAlignment = false) {
```

Comment:
I think I would prefer calling that `IsUnionMember` or something like that (and reverse the condition everywhere)

Reply:
Yeah,I agree with Corentin, but mostly because it is from the wrong 'direction'.  Bools like this should be named in a way that makes it easy for a user of this function, with no understanding of the implementation to understand.

So it should be, "I want to know if these two are layout compatible... Why would I ignore alignment?" vs "Oh, I'm not dealing with a union member, so I can set this false!".

Reply:
Exactly!

Reply:
> but mostly because it is from the wrong 'direction'

This is spot-on observation. I wasn't sure which direction should I look at this from, but seeing both of you agree, I'm happy to change this.